Sources/MapboxDirections/MapboxStreetClass.swift

```diff
@@ -0,0 +1,65 @@
+
+import Foundation
+
+public enum MapboxStreetClass: String, Codable {
```

Comment:
Unlike `RoadClasses`, Mapbox Streets V8 assigns a single class to an intersection. Also, possible list of Mapbox road classes is not compatible with exiting `RoadClasses`, so I decided to put it as a separate `enum`.

Reply:
Yes, that also makes sense because you can’t request a route that only traverses secondary roads, for example.